src/components/IntegrationsView/CustomFieldsAndWidgets.tsx

```diff
@@ -79,16 +111,41 @@ export const CustomLabel = ({
 
 export const CustomDescriptionField = ({
   children = "",
+  mb = "2",
 }: {
   children?: string;
+  mb?: string;
 }) => {
   return (
-    <Text size="1" mb="2" style={{ display: "block", opacity: 0.85 }}>
+    <Text
+      size="1"
+      mb={{
+        initial: "0",
+        xs: mb,
+      }}
+      style={{ display: "block", opacity: 0.85 }}
+    >
       <Markdown>{children}</Markdown>
     </Text>
   );
 };
 
+export const CustomBoolField = ({
+  id,
+  name,
+  defaultValue,
+}: {
+  id: string;
+  name: string;
+  defaultValue: boolean;
+}) => {
+  return (
+    <Box>
```

Comment:
note to self: take a look at this and see if 
<Box> could be `<Label>`, so a label can also be displayed if needed.

Reply:
DataList is being used for the labels.